client/state/data-layer/wpcom/sites/stats/google-my-business/test/index.js

```diff
@@ -11,6 +11,48 @@ import { fetchStats, receiveStats, fromApi } from '..';
 import { receiveGoogleMyBusinessStats } from 'state/google-my-business/actions';
 import { http } from 'state/data-layer/wpcom-http/actions';
 
+describe( '#fromApi', () => {
+	const data = {
+		time_zone: 'Europe/London',
+		metric_values: [
+			{
+				metric: 'QUERIES_DIRECT',
+				total_value: {
+					time_dimension: {
+						timeRange: {
+							endTime: '2018-04-19T23:59:59.900Z',
+							startTime: '2018-04-13T00:00:00Z',
+						},
+					},
+					metric_option: 'AGGREGATED_TOTAL',
+					value: 0,
+				},
+			},
+		],
+	};
+
+	test( 'should transform snake_case to camelCase', () => {
```

Comment:
The `convertToCamelCase` helper already has a unit test, so maybe we can remove this one completely.